src/main/java/core/basesyntax/dao/FruitDao.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.Fruit;
+import java.util.Map;
+
+public interface FruitDao {
+    Integer getBalance(Fruit fruit);
+
+    Integer add(Fruit fruit, Integer amount);
+
+    Map<Fruit, Integer> getStorage();

```

Comment:
```suggestion
    Integer get(Fruit fruit);

    Integer update(Fruit fruit, Integer amountToAdd);

    Map<Fruit, Integer> getAll();
```
these are preferable method names for dao layer